lldb/source/Plugins/TypeSystem/Swift/SwiftASTContext.cpp

```diff
@@ -1268,6 +1268,11 @@ static const char *getImportFailureString(swift::serialization::Status status) {
   case swift::serialization::Status::TargetTooNew:
     return "The module file was built for a target newer than the current "
            "target.";
+  case swift::serialization::Status::SDKMismatch:
+    return "The module file was built with a different SDK than the one in use "
+           "to build the client.";
+  case swift::serialization::Status::RevisionIncompatible:
+    return "The precise revision version doesn't match.";
```

Comment:
Should this string also mention "module file"? It's very generic otherwise.

Reply:
It seemed like all of these comments were copying the docstring from enum declaration, so I followed that. But yes and thanks, this message can be better.